build-system/global-configs/prod-config.json

```diff
@@ -8,7 +8,6 @@
   "expDoubleclickA4A": 0.1,
   "a4aProfilingRate": 1,
   "amp-form-custom-validations": 1,
-  "link-url-replace": 1,
```

Comment:
TIL: we should have waited until the code changes are in prod before removing the flag. (since release process does not have a nice way of cherry picking config changes). It is fine though, manually cherry picking to ensure this does not make it to prod yet.